exampleCourse/questions/element/unitsInput/question.html

```diff
@@ -1,118 +1,102 @@
 <pl-question-panel>
     <p>
         Questions below were designed to showcase the different features of
-        <a
-            href="http://prairielearn.readthedocs.io/en/latest/elements/#pl-units-input-element"><code>pl-units-input</code></a>.
-        The goal of <a
-            href="http://prairielearn.readthedocs.io/en/latest/elements/#pl-units-input-element"><code>pl-units-input</code></a>
+        <a href="http://prairielearn.readthedocs.io/en/latest/elements/#pl-units-input-element"><code>pl-units-input</code></a>.
+        The goal of <a href="http://prairielearn.readthedocs.io/en/latest/elements/#pl-units-input-element"><code>pl-units-input</code></a>
```

Comment:
Nit: I know this was already written this way, but not need to have the same link twice in the same paragraph.
```suggestion
        The goal of <code>pl-units-input</code>
```